src/services/services.ts

```diff
@@ -2986,6 +3006,7 @@ namespace ts {
 
             function tryGetGlobalSymbols(): boolean {
                 let containingObjectLiteral = getContainingObjectLiteralApplicableForCompletion(contextToken);
+                let jsxElement: JsxElement, jsxSelfClosingElement: JsxSelfClosingElement;

```

Comment:
Use another `let` declaration, we don't generally declare variables like this.

Are these ever uesd?